Gems/Atom/RPI/Code/Source/RPI.Builders/Model/ModelExporterComponent.h

```diff
@@ -55,7 +55,7 @@ namespace AZ
 
                 AZStd::string_view m_relativeFileName;
                 AZStd::string_view m_extension;
-                const Uuid m_sourceUuid;
+                const Uuid m_sourceUuid=Uuid::CreateNull();
```

Comment:
nitpick missing space around the `=` sign

Reply:
Fixed in 844cc43